jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -30,12 +30,16 @@ pipeline {
 
     stages {
         stage('Set build name') {
-            when {
-                not {environment name: 'GIT_BRANCH', value: 'origin/master'}
-            }
```

Comment:
Just for the sake of it, I've tried with:
```
when {
    not {
        anyOf {
            environment name: 'ghprbPullTitle', value: null
            environment name: 'ghprbPullLink', value: null
        }
    }
}
```
and it does compare against `null` values. I personally think this looks cleaner, what do you think? 🤔

Reply:
Agreed, PR updated.